administration/package.json

```diff
@@ -46,6 +46,9 @@
     "@babel/core": "^7.22.5",
     "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
     "@bufbuild/protoc-gen-es": "^1.2.1",
+    "@eslint/compat": "^1.2.7",
+    "@eslint/eslintrc": "^3.3.0",
+    "@eslint/js": "^9.21.0",
```

Comment:
:x: Please remove, see comment above for `@eslint/js`. For `@eslint/eslintrc` from the [docs](https://www.npmjs.com/package/@eslint/eslintrc):
> This repository contains the legacy ESLintRC configuration file format for ESLint. This package is not intended for use outside of the ESLint ecosystem. It is ESLint-specific and not intended for use in other programs.
Note: This package is frozen except for critical bug fixes as ESLint moves to a new config system.

Reply:
Not all configs are supported yet by Eslint v9 look at : https://github.com/eslint/eslint/issues/18391
so I think we need a compatibility tool ..
https://eslint.org/docs/latest/use/configure/migration-guide#using-eslintrc-configs-in-flat-config

Reply:
Do we actually need both fixupConfigRules as well as FlatCompat to support legacy configs? Sounds to me like this is a duplication :thinking: